lib/huginn_acumen_order_agent/acumen_client.rb

```diff
@@ -20,12 +20,12 @@ def get_invoice_details(invoice_ids)
   end
 
   def execute_query(body, headers)
-      response = @faraday.run_request(:post, "#{@auth['endpoint']}/Query", body, headers)
+      response = @faraday.run_request(:post, "#{@auth['endpoint']}Query", body, headers)
```

Comment:
I know we talked about this being ok, but in hindsight, I think we actually do want to change it.

We should have some logic around setting `endpoint` that ensures it ends in a `/`.

Reply:
I think I fixed this now. Let me know what you think

Reply:
@mrogers-5s Fixed